src/python/pants/option/alias_test.py

```diff
@@ -105,8 +105,8 @@ def test_no_expand_when_no_aliases() -> None:
             pytest.raises(
                 CliAliasCycleError,
                 match=(
-                    r"CLI alias cycle detected in `\[cli\]\.alias` option: "
-                    r"other-alias -> cycle -> other-alias"
+                    r"CLI alias cycle detected in `\[cli\]\.alias` option:"
+                    + r"other-alias -> cycle -> other-alias"
```

Comment:
Now there's no space after the colon.. I guess there should be a newline instead?